Formula/biobloomtools.rb

```diff
@@ -0,0 +1,34 @@
+class Biobloomtools < Formula
+  # cite Chu_2014: "https://doi.org/10.1093/bioinformatics/btu558"
+  desc "BioBloom Tools (BBT): Bloom filter for bioinformatics"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/biobloomtools/"
+  url "https://github.com/bcgsc/biobloom/releases/download/2.0.13/biobloomtools-2.0.13.tar.gz"
+  sha256 "7b4aeef70feb3fc31db2f4695159523272eadd8787b33c2902e2970a7d569ba3"
+  revision 1
+
+  head do
+    url "https://github.com/bcgsc/biobloom.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  fails_with :clang # needs openmp
+
+  depends_on "boost" => :build
+  depends_on "gcc" if OS.mac? # for openmp
+
+  def install
+    system "./autogen.sh" if build.head?
+    system "./configure",
+      "--disable-debug",
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "Usage", shell_output("#{bin}/biobloommaker --help")
+    assert_match "Usage", shell_output("#{bin}/biobloomcategorizer --help")
```

Comment:
Change both `--help` to `--help 2>&1` to redirect standard error to standard output.